pkg/nuclide-source-control-helpers/lib/hg-repository.js

```diff
@@ -35,7 +35,7 @@ export default function findHgRepository(startDirectoryPath: string): ?HgReposit
       }
       return {repoPath, originURL, workingDirectoryPath};
     }
-    const parentDir = nuclideUri.join(workingDirectoryPath, '..');
+    const parentDir = nuclideUri.join(workingDirectoryPath);
```

Comment:
This should be .dirname bc removing the second argument from the join call makes it no-op and won't detect mercurial repositories